frontend/test/metabase/scenarios/models/model-actions.cy.spec.js

```diff
@@ -160,9 +160,9 @@ describe(
       });
 
       fillActionQuery(QUERY);
+      cy.contains("New Action").type("{selectAll}{backspace}Discount order");
```

Comment:
We try to use the accessible selectors (`.findByText/Label/etc`) from cypress wherever possible.
The cypress `clear()` function is just a bit easier to read. https://docs.cypress.io/api/commands/clear

```suggestion
      cy.findByText("New Action").clear("Discount order");
```

Reply:
Thanks for the suggestions, updated the commit with them. Still getting used to Cypress coming from Playwright 🙂

Reply:
👍 just FYI, since we squash-merge everything into master, there's no need to amend + force-push commits. Sometimes it makes re-reviewing easier if you keep follow-up commits separate.

Reply:
Makes sense, I'll make sure to just add another commit next time